org.tuxpaint.Tuxpaint.appdata.xml

```diff
@@ -24,7 +24,14 @@ SentUpstream: 2014-09-18
       help them be creative.
     </p>
   </description>
+  <developer_name>New Breed Software et al.</developer_name>
   <url type="homepage">http://tuxpaint.org/</url>
+  <url type="bugtracker">https://sourceforge.net/p/tuxpaint/_list/tickets</url>
+  <url type="faq">http://www.tuxpaint.org/docs/en/html/FAQ.html</url>
+  <url type="help">http://www.tuxpaint.org/docs/</url>
+  <url type="donation">http://www.newbreedsoftware.com/donate/</url>
+  <url type="translate">http://www.tuxpaint.org/help/po/</url>
+  <url type="contact">http://www.tuxpaint.org/contact/</url>
```

Comment:
This covers the complete list of [available url types in the appstream spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url).